packages/core/content-type-builder/admin/src/translations/en.json

```diff
@@ -60,7 +60,7 @@
   "error.contentTypeName.reserved-name": "This name cannot be used in your project as it might break other functionalities",
   "error.validation.enum-duplicate": "Duplicate values are not allowed",
```

Comment:
What do you think of : `Duplicate values are not allowed (special characters are not counted)` ?

Reply:
I'd go for: "Duplicate values are not allowed (only alphanumeric characters are taken into account)"

Reply:
Done in 87435f5593e8395dc67592997c6e81efdb21bdef